src/test_utils/mod.rs

```diff
@@ -28,26 +29,33 @@ pub fn policy() -> Result<Policy, Error> {
         ],
         false,
     );
-    let mut policy = Policy::new();
-    policy.add_dimension(sec_level)?;
-    policy.add_dimension(department)?;
-    Ok(policy)
+    let _policy = Policy::new();
```

Comment:
There should be no need for that anymore. Plus, you are ignoring it anyway. Prepending `_` to variable names is dangerous as it deactivates `clippy` lints. Only use it when you have no other choice.